packages/next-frontend/src/components/create-campaign/form/pool-step/preview/index.tsx

```diff
@@ -24,12 +24,12 @@ export function PoolStepPreview({ pool }: PoolStepPreviewProps) {
                     }}
                 />
                 <div className={styles.poolInfo}>
-                    <Typography weight="medium" variant="sm">
+                    <Typography weight="medium" variant="lg">
                         {pool.token1.symbol} / {pool.token0.symbol}
                     </Typography>
                     {pool.fee && (
                         <Typography variant="xs" light>
-                            {numeral(pool.fee / 10_000).format("0%")}
+                            {pool.fee / 10_000}%
```

Comment:
I'm not super sure this is right as you have some dexes that work with bips, some with hundredths of a bip etc etc. I think it must be handled on the backend.

Reply:
Oh if that's the case yes, we probably need to handle this in the backend.